module1/src/com/epam/linearprograms/HomeTask_1_3.java

```diff
@@ -0,0 +1,16 @@
+package com.epam.linearprograms;
+
+public class HomeTask_1_3 {
+    public static void main(String[] args) {
+        double x = 2.0;   //Значение в радинах
+        double y = 2.0;   //Значение в радинах
+        double z;
+        if ( Math.cos(x)-Math.sin(y)==0){
```

Comment:
delete space after (
```suggestion
        if (Math.cos(x) - Math.sin(y) == 0) {
```